### PR TITLE
Add basic web3.js document

### LINKED ISSUE #604: Add web3.js Guide and snippets
Use https://github.com/onflow/flow-evm-gateway/blob/gregor/dev/integration/web3js/index.js
to create a simple Guide under EVM/Guides/Clients for using web3.js

### PR DESCRIPTION
**Pull request by jribbink**
closes #604

### CODE REVIEW

File: docs/evm/tools/clients/web3-js.md [hunk 1]
```diff
@@ -6,20 +6,88 @@ sidebar_position: 2
...
+    ... // ABI of deployed contract
+]
+
+const contractAddress
+const contract = 
+```
+
+Methods on the smart contract can now be called in the following ways.
+
+#### Querying state
```
Comment (chasefleming) [inline docs/evm/tools/clients/web3-js.md:72]: TItlecase. "Querying State"

File: docs/evm/tools/clients/web3-js.md [hunk 2]
```diff
@@ -6,20 +6,88 @@ sidebar_position: 2
...
+
+We can query data from the contract by using the `call` function on one of the contract's methods.  This **will not** mutate the state and **will not** send a transaction.
+
+```js
+const result = contract.methods.someGetterFunction().call()
+
+console.log(result)
+```
+
+#### Mutating state
```
Comment (chasefleming) [inline docs/evm/tools/clients/web3-js.md:82]: Same on casing

File: docs/evm/tools/clients/web3-js.md [hunk 3]
```diff
@@ -6,20 +6,200 @@ sidebar_position: 2
...
 
-web3.js is a comprehensive JavaScript library designed to facilitate interaction with EVM-compatible blockchain networks. Its integration with FlowEVM will enable developers to deploy, interact with smart contracts, and perform various blockchain operations within the Flow ecosystem seamlessly.
+## Installation
 
-Stay tuned for detailed guides on:
+:::info
+This guide assumes you have the latest version of [Node.js](https://nodejs.org/en) installed.
+:::
+
+To install `web3`, run the following command:
```
Comment (gregsantos) [inline docs/evm/tools/clients/web3-js.md:20]:
```suggestion
To install `web3` in your project, run the following command:
```

File: docs/evm/tools/clients/web3-js.md [hunk 4]
```diff
@@ -6,20 +6,200 @@ sidebar_position: 2
...
+npm install web3
+```
+
+## Initializing Web3 With Flow
 
-- Installing and setting up web3.js for your projects.
-- Connecting to the FlowEVM network using web3.js.
-- Reading and writing data to the blockchain.
-- Deploying and interacting with smart contracts on FlowEVM.
+To use `web3` in your project, start by importing the module and initializing your `web3` instance with the desired Flow RPC endpoint.
```
Comment (gregsantos) [inline docs/evm/tools/clients/web3-js.md:28]:
```suggestion
To use `web3` in your project, start by importing the module and initializing your `web3` instance with the desired Flow RPC endpoint. We are specifying Flow Previewnet in this example.
```

File: docs/evm/tools/clients/web3-js.md [hunk 5]
```diff
@@ -6,20 +6,200 @@ sidebar_position: 2
...
+console.log(gasPrice) // Gas price in attoFlow
+```
+
+For more information about other queries you can make `web3`, please see the [official documentation](https://docs.web3js.org/).
+
+## Interacting With Smart Contracts
+
+The `web3` library allows developers to interact with smart contracts via the `web3.eth.Contract` API.
+
+For this example we will use the following `Storage` contract, deployed on the Flow Previewnet to the address `0x4c7784ae96e7cfcf0224a95059573e96f03a4e70`.  Note that anybody can interact with this contract, as it is deployed on a public network, so state may not always be as expected.
```
Comment (gregsantos) [inline docs/evm/tools/clients/web3-js.md:70]:
```suggestion
For this example we will use the following `Storage` contract, deployed on the Flow Previewnet to the address `0x4c7784ae96e7cfcf0224a95059573e96f03a4e70`.  Note that anyone can interact with this contract, as it is deployed on a public network, so state may not always be as expected.
```

**Review by gregsantos — APPROVED** [review]
couple nits but this is great!

**Pull request merged**